spec/controllers/split_checkout_controller_spec.rb

```diff
@@ -165,6 +165,7 @@
 
           expect(order.state).to eq "confirmation"
           expect(order.payments.first.adjustment.amount).to eq 1.23
+          expect(order.payments.first.amount).to eq order.item_total + order.adjustment_total
```

Comment:
The other expectations here just verified that the fees exist and that order total is correct. This additionally verifies that the payment object's amount includes relevant fees, so they'll be included in the total that's considered "paid" when the order is completed. The payment fee was not included in that amount previously, leaving the payment state as "balance due".